### ISSUE TITLE
Indentation fails

### ISSUE THREAD

**Issue by introom** [skuro/plantuml-mode#78]
After the latest update, a line will be pushed to the very left when pressing the return key.

I suspect it's due to the incorporation of indentation stuff.

**Comment by introom** [comment]
I add a picture as a supplementary.

If you start from scratch, you will end up have this:

![image](https://user-images.githubusercontent.com/669045/44628554-527ca100-a974-11e8-8629-23f8ea181c47.png)

**Comment by dinuscxj** [comment]
``` plantuml
@startuml
   package "kuaishow-common-proto" {
           component Kwai
           component KwaiGo
           component AcFun
   }

   package "kwaigo-log-proto" {
           [KwaiGo] ..> [KwaiGo Client->Data] : 拆分
   }

   package "kwaigo-api-proto" {
           [KwaiGo] ..> [KwaiGo Client->Server] : 拆分
   }

   cloud "log server\n" {
   [KwaiGo Client->Data] -->[KwaiGo Log Server] : 上传
   }

   cloud "api server\n" {
   [KwaiGo Api Server]
   }

   [KwaiGo Client->Server] --> [KwaiGo Api Server] : 请求
   [KwaiGo Api Server] --> [KwaiGo Client->Server] : 响应

   database "LogSql" {
   folder "Log" {
   [ReportEvent]
   }
   frame "event" {
   [EventPackage]
   }

   frame "stat" {
   [StatPackage]
   }
   }

   [KwaiGo Log Server] --> [ReportEvent] : 解析存储
   [ReportEvent] --> [EventPackage]
   [ReportEvent] --> [StatPackage]

   database "ApiSql" {
   folder "kwaigo" {
   [KwaiGoData]
   }
   frame "user" {
   [UserData]
   }

   frame "photo" {
   [PhotoData]
   }
   }

   [KwaiGo Api Server] --> [KwaiGoData] : 请求
   [KwaiGoData] --> [KwaiGo Api Server] : 计算

   [KwaiGoData] --> [UserData]
   [KwaiGoData] --> [PhotoData]
   @enduml
```

Only package work

**Comment by skuro** [comment]
The current strategy to indent code is based on regexes, which were not including the keywords you used up there. The latest master includes a fix that adds a few keywords (see 56d668c). Not a perfect solution, but should work for the time being.
